scipy/stats/tests/test_continuous_basic.py

```diff
@@ -59,26 +59,26 @@
                  'levy_stable', 'rv_histogram_instance']
 
 # skip check_fit_args_fix (test is slow)
```

Comment:
You might want to add to `distslow` if `test_cont_basic` isn't going to get faster.

Reply:
alright, let's see now that argcheck is vectorized natively in numpy, else, will do so

Reply:
`2.02s call     tests/test_continuous_basic.py::test_cont_basic[genhyperbolic-arg33]`

This is the current situation, is this too slow? Othe ghyp tests are faster

Reply:
Yes, that would be longer than any other distribution, I think.  I think we're all set now.